packages/fhir-converter/src/templates/cda/Sections/Payers.hbs

```diff
@@ -55,6 +55,12 @@
                           {{>References/Coverage/beneficiary.hbs ID=(generateUUID (toJsonString payerEntry.act)) canBeUnknown=true  }},
                     {{/if}}
                 {{/each}}
+                {{#each (toArray payerEntry.act.participant) as |participantEntry|}}
+                    {{#if participantEntry.participantRole.id.extension}}
+                        {{>References/Coverage/subscriberId.hbs subscriberId=participantEntry.participantRole.id.extension ID=(generateUUID (toJsonString payerEntry.act))}},
+                        {{>References/Coverage/identifier.hbs payerEntry=participantEntry.participantRole ID=(generateUUID (toJsonString payerEntry.act))}},
```

Comment:
The way the converter works is that it will merge the properties of the objects returned by each of the `.hbs` template?

Reply:
yea, so the `identifier` field that's generated with `>Resource/Coverage.hbs` will NOT be overwritten by the `>References/Coverage/identifier.hbs`. The arrays will be combined